packages/block-library/src/classic/convert-to-blocks-button.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { BlockControls } from '@wordpress/block-editor';
+import { Toolbar, ToolbarButton } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { rawHandler, serialize } from '@wordpress/blocks';
+
+const ConvertToBlocksButton = ( { clientId } ) => {
+	const { replaceBlocks } = useDispatch( 'core/block-editor' );
+	const block = useSelect( ( select ) => {
+		return select( 'core/block-editor' ).getBlock( clientId );
+	} );
```

Comment:
```suggestion
	}, [ clientId ] );
```

The dependencies should help in terms of performance. (Avoid running the callback on each render)